playground/black_scholes_explicit.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+class BlackScholesExplicitSolver():
+
+    # using call option as an example for now - will update it to toggle between put/call
+
+    def __init__(self, S_max, expiry, sigma, r, K, s_nodes, t_nodes = None):
+        self.__S_max = S_max
+        self.__expiry = expiry
+        self.__sigma = sigma
+        self.__r = r
+        self.__K = K
+        self.__s_nodes = s_nodes
+        self.__t_nodes = t_nodes
+        self.__V = None
+
+    def solve(self):
+
+        if self.__t_nodes is None:
```

Comment:
following our conversation in PR #9 where we discussed the possibility of allowing users to define their own t_nodes:

would this solution work?